src/Mail/SmtpMailer.php

```diff
@@ -58,6 +61,7 @@ public function __construct(array $options = array())
 		if (!$this->port) {
 			$this->port = $this->secure === 'ssl' ? 465 : 25;
 		}
+		$this->context = isset($options['context']) ? $options['context'] : array();
```

Comment:
Maybe `$this->context = isset($options['context']) ? $options['context'] : NULL;`